### ISSUE TITLE
Allow container sizes to be determined by generators

### ISSUE THREAD

**Issue by jupp0r** [emil-e/rapidcheck#143]
Container sizes can either be `nonEmpty`, completely random, fixed or determined by `suchThat`. I 'd like to have a random range of container sizes.

**Comment by emil-e** [comment]
Yes, this would indeed be a useful feature but I rarely have time to work on RapidCheck these days. I do welcome a PR though, and I will spend time reviewing it.
